tests/test_assessment_endpoints.py

```diff
@@ -51,27 +51,6 @@ def test_create_duplicate_application_assessment_fails(
             == "An assessment for this application already exists"
         )
 
-    def test_get_assessments_list(self, flask_test_client):
```

Comment:
just curious! why are we getting rid of this test? 🤔

Reply:
there will never be a situation where one wants every assessment, so I'm not sure if the endpoint is needed at all?